default/scripting/techs/defense/Defense.focs.txt

```diff
@@ -28,6 +28,15 @@ Tech
                 SetDefense value = min(Value + [[PLANET_DEFENSE_FACTOR]], Value(Target.MaxDefense))
                 SetTroops value = min(Value + 1, Value(Target.MaxTroops))
             ]
+
+        EffectsGroup    // set troops to 6 if just colonized
```

Comment:
specific value in comment could be inaccurate if code is changed elsewhere...

Reply:
How about "ensure troop defense on a newly colonized planet"

Reply:
whatever you like, just remove the "6"

Reply:
Fixed.